src/vc_util/src/lib.rs

```diff
@@ -372,8 +372,16 @@ fn validate_claims_match_spec(
     let subject = Subject::from_json_value(credential_subject.clone()).map_err(|_| {
         inconsistent_jwt_claims("missing credentialSubject in VerifiedAdult JWT vc")
     })?;
+    let credential_type = &spec.credential_type;
```

Comment:
Shouldn't this now enable _generic_ validation? I.e. this function should not specifically validate anything related to "adult", right?

Reply:
Yes, this does enable generic validation, which will come in a follow-up PR.